Lib/fontTools/varLib/__init__.py

```diff
@@ -92,10 +92,9 @@ def _add_fvar_avar(font, axes, instances):
 	for axis in axes.values():
 		curve = avar.segments[axis.tag] = {}
 		if not axis.map or all(k==v for k,v in axis.map.items()):
-			continue
-		interesting = True
-
-		items = sorted(axis.map.items())
+			interesting |= False
```

Comment:
It was supposed to make sure segment entries are being added even if not different from the default mapping after the first neccessary segment map has been found. It was a workaround for the DirectWrite bug.

I see now it's buggy …